Task 1/Even Odd/script.js

```diff
@@ -0,0 +1,12 @@
+let input = document.querySelector("input")
+let displayText = document.querySelector(".display-text")
+
+input.addEventListener("input", inputType)
+
+function inputType() {
+    if(input.value % 2 == 0) {
+       displayText.textContent = "The number you typed is an even number"
+    } else {
+        displayText.textContent = "The number you typed is an odd number"
+    }
+}
```

Comment:
A newline at end of file is largely followed coding practice! Maybe we can have it here?
```suggestion
}

```